documentation/requirements.txt

```diff
@@ -4,4 +4,5 @@ sphinx-rtd-theme
 cairosvg
 breathe
 sphinxcontrib-trio
+sphinxcontrib-makedomain
```

Comment:
It's possible this will fail due to https://github.com/sphinx-contrib/domaintools/issues/3...